server/src/test/java/io/spine/server/entity/TransactionShould.java

```diff
@@ -69,7 +69,33 @@
 
     @Rule
     public ExpectedException thrown = ExpectedException.none();
-    
+
+    private static ValidationException validationException() {
+        ValidationException ex = new ValidationException(Lists.newLinkedList());
+        return ex;
+    }
+
+    private static List<ConstraintViolation> someViolations() {
+        ConstraintViolation expectedViolation = ConstraintViolation
+                .newBuilder()
+                .setMsgFormat("Some violation %s")
+                .addParam("1")
+                .build();
+        return newArrayList(expectedViolation);
+    }
+
+    private static boolean checkPhase(Event event, Phase phase, boolean successful) {
+        Message eventMessage = Events.getMessage(event);
+        return eventMessage.equals(phase.getEventMessage())
+                && event.getContext()
+                        .equals(phase.getContext())
+                && phase.isSuccessful() == successful;
```

Comment:
Please split the expression into 3 with 3 local variables to make it more debuggable.